ProffieOS.ino

```diff
@@ -461,6 +461,7 @@ struct is_same_type<T, T> { static const bool value = true; };
 #include "transitions/wave.h"
 #include "transitions/select.h"
 #include "transitions/extend.h"
+#include "transitions/center-wipe.h"
```

Comment:
no dashes in file names please

Reply:
sorry, typo, file was underscore, fixed include.

Reply:
Don't you test these things?